lib/addons/jupyterhub/index.ts

```diff
@@ -176,6 +175,8 @@ export class JupyterHubAddOn extends HelmAddOn {
         setPath(values, "ingress.enabled", enableIngress);
 
         if (enableIngress){
+            const dep = clusterInfo.getScheduledAddOn(AwsLoadBalancerControllerAddOn.name);
```

Comment:
is this related to the order of how dependencies are added? I am a bit unclear to why change from dependable to scheduled addon?

Reply:
I just realized this change is not necessary. Will remove this. (still need to run dependable decorator for NLB)